sklearn/metrics/_classification.py

```diff
@@ -1201,10 +1201,10 @@ def fbeta_score(
     The F-beta score is the weighted harmonic mean of precision and recall,
     reaching its optimal value at 1 and its worst value at 0.
 
-    The `beta` parameter determines the weight of recall in the combined
-    score. ``beta < 1`` lends more weight to precision, while ``beta > 1``
-    favors recall (``beta -> 0`` considers only precision, ``beta -> +inf``
-    only recall).
+    The `beta` parameter represents the ratio of recall to
+    precision. `beta > 1` gives more weight to recall, while
```

Comment:
Thinking about it more, what do you think of this rephrasing?

```suggestion
    The `beta` parameter represents the relative importance of
    recall to precision. `beta > 1` gives more weight to recall, while
```

Reply:
> Thinking about it more, what do you think of this rephrasing?

I personally think `ratio` is more intuitive - a plain old mathematical concept that brings the idea of fractions to the reader's mind. While `relative importance` - despite definitely valid - I feel it could confuse some readers, they may be unsure if it refers to a new or more complex concept.

> I was trying to think of a way not to say importance twice. In any case, I am okay with your original suggestion:
> 
> > ratio of recall importance to precision importance

same here! couldn't find one.. but I still consider it a small price for the sake of being crystal clear : )


as you were ok with the original suggestion, I'm editing it. If you want to think more about it or get other people's opinion, that's totally cool, will be waiting for the approval or new comments. cheers!